pom.xml

```diff
@@ -68,6 +68,8 @@
         <module>security/keycloak-oidc-client</module>
         <module>sql-db/sql-app</module>
         <module>sql-db/multiple-pus</module>
+        <module>lifecycle-application</module>
```

Comment:
This seems to have snuck in erroneously.

Reply:
This was missing at first, that's why I added it here.